### PR TITLE
feat/AB#66444_dashboard-button-actions

### PR DESCRIPTION
**Pull request by matheus-relief**
# Description
This PR adds the possibility to add button actions on dashboards

## Ticket
[DB - Ability to create buttons on top of pages](https://dev.azure.com/WHOHQ/EMSSAFE/_boards/board/t/App%20Builder%20-%20WHE%20Dashboard/Stories?text=66444&System.AssignedTo=%40me)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvement (refactor or addition to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

See screenshots

## Sreenshots

Peek 2023-05-29 13-51

# Checklist:

( * == Mandatory ) 

- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/models/dashboard.model.ts [hunk 1]
```diff
@@ -17,6 +24,15 @@ const dashboardSchema = new Schema<Dashboard>(
     name: String,
     structure: mongoose.Schema.Types.Mixed,
     showFilter: Boolean,
+    buttons: [
```
Comment (AntoineRelief) [inline src/models/dashboard.model.ts:27]: @matheus-relief 
better create submodel there

File: src/schema/mutation/editDashboard.mutation.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export default {
     structure: { type: GraphQLJSON },
     name: { type: GraphQLString },
     showFilter: { type: GraphQLBoolean },
+    buttons: { type: GraphQLJSON },
```
Comment (AntoineRelief) [inline src/schema/mutation/editDashboard.mutation.ts:26]: @matheus-relief 
there we can create a custom input type

File: src/models/dashboard.model.ts (hunk 1 above)
Comment (matheus-relief) [reply src/models/dashboard.model.ts]: done

File: src/schema/mutation/editDashboard.mutation.ts (hunk 2 above)
Comment (matheus-relief) [reply src/schema/mutation/editDashboard.mutation.ts]: done

**Pull request merged**